docs/compilation/multi_precision.md

```diff
@@ -1,6 +1,10 @@
 # Multi Precision
 
-Each integer in the circuit has a certain bit-width, which is determined by the inputset. These bit-widths can be observed when graphs are printed:
+This document explains the multi-precision option for bit-width assignment for integers. Multi-precision bit-width assignment allows for flexible bit-width allocation in a circuit to optimize the efficiency and ensure the compatibility with Fully Homomorphic Encryption (FHE). 
```

Comment:
I don't understand the second sentence here... If it's a summary of what the option is it should be something like:
```
The multi-precision option enable the fronted to use the smallest bit width possible for each FHE operation.  
```

Reply:
Maybe removing the notion of compatibility its obvious and let believe that if that option is not use we could generate non-compatible operations.